cypress/support/index.ts

```diff
@@ -1,10 +1,15 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { LocalStorageProvider } from "../../src/api/local-storage-provider.model";
 import "./commands";
 
 Cypress.Screenshot.defaults({
   capture: "fullPage",
   disableTimersAndAnimations: true,
   scale: true,
 });
+
+beforeEach(() => {
+  new LocalStorageProvider().reloadDb();
```

Comment:
To nie może być w pełni globalne, bo będzie się też niepotrzebnie wykonywać dla unit testów

Reply:
Na razie próbuję naprawić testy które nie przechodzą 
Przywróciłem to
To była pierwsza próba i ona nie pomogła
Teraz nastał kolej pielęgniarki

Reply:
![image](https://user-images.githubusercontent.com/21079319/110514595-c1234380-8107-11eb-8740-07ebbd941983.png)